src/content/documentation/develop/web-ext-command-reference.md

```diff
@@ -63,73 +64,98 @@ This page lists all the commands and options available under the [web-ext](https
 
 web-ext has the following commands; options specific to these commands are included as subsections.
 
-<section id="web-ext-build"></section>
+<section id="web-ext-build">
 
 ### `web-ext build`
 
 Packages an extension into a `.zip` file, ignoring files that are commonly unwanted in packages, such as `.git` and other artifacts. The name of the `.zip` file is taken from the [name](https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/manifest.json/name) field in the extension [manifest](https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/manifest.json).
 
```

Comment:
```suggestion
</section>

```

Reply:
This section continues until the line 90 as it contains all the options for that command.

Reply:
ah oops, sorry I thought I had removed this suggestion.